### PR TITLE
Update Action Icon style

### PR DESCRIPTION
**Pull request by almonais**
## Catégorisation
- :bug: Cette PR est une correction de bug

## Description
Maj Action Icon
image

-----

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1038

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1038

File: packages/scss/src/components/_action-icon.scss [hunk 1]
```diff
@@ -1,7 +1,7 @@
 .actionIcon {
 	background: transparent;
 	border: 0;
-	border-radius: 50%;
+	border-radius: _component("textfield.border-radius");
```
Comment (sanlucca) [inline packages/scss/src/components/_action-icon.scss:4]: It's weird to use a radius coming from another component for no reason.
Why not use either the commons.border.radius or define a specific actionIcon border radius ?

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1038

**Pull request merged**